src/service/address/address.ts

```diff
@@ -251,68 +423,46 @@ export class Address {
       })
   }
 
-  private async getFullBalance(addresses: string[]): Promise<{nativeBalance: string, tokenBalances: []}> {
-    const network = this.config.network
-    switch (true) {
-      case [Network.TRON, Network.TRON_SHASTA].includes(network):
-        if (addresses.length !== 1) {
-          throw new Error(`UTXO based networks like ${network} support only one address per call.`)
-        }
-        return this.connector
-          .get<{
-            balance: number,
-            createTime: number
-            trc10: [{
-              key: string,
-              value: number,
-            }]
-            trc20: [{[key: string]: string}]
-            freeNetLimit: number,
-            bandwidth: number,
-          }>({
-            path: `tron/account/${addresses[0]}`,
-          })
-          .then(async (r) =>
-          {
-            return Object.create({
-              nativeBalance: r.balance.toString(),
-              tokenBalances: r.trc20.length > 0 ? await this.processTRC20TokenBalanceDetails(r.trc20) : [],
-            })
-          })
-    }
-    throw new Error(`Unsupported network ${network} for now.`)
-  }
-
   private async getNativeBalance(addresses: string[]): Promise<string[]> {
     const network = this.config.network
     switch (true) {
       case isEvmBasedNetwork(network):
         return Promise.all(
```

Comment:
why no `await` :(

Reply:
Reverted the code back as it was few commits before. It should be now more readable